docs/tutorials/namespace-tutorial.md

```diff
@@ -0,0 +1,105 @@
+# Namespaces
+## Introduction
+The following workflow will help to get familiar when working with namespaces. It assumes Epinio has been previously installed. Should not be the case. please visit [Install Epinio](docs/installation/install_epinio.md) to achieve it.
```

Comment:
While the new text was added, see line 5, this line was not removed.

Reply:
Ah, thank you. Addressed in b6d63b5cd